modules/virt-example-nmstate-multiple-interfaces.adoc

```diff
@@ -7,26 +7,40 @@
 
 You can create multiple interfaces in the same node network configuration policy. These interfaces can reference each other, allowing you to build and deploy a network configuration by using a single policy manifest.
 
-The following example snippet creates a bond that is named `bond10` across two NICs and a Linux bridge that is named `br1` that connects to the bond.
+The following example snippet creates a bond that is named `bond10` across two NICs and VLAN that is named `bond10.103` that connects to the bond.
 
 [source,yaml]
 ----
-...
+apiVersion: nmstate.io/v1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: bond-vlan
+spec:
+  nodeSelector:
+    kubernetes.io/hostname: dell-per740-14.rhts.eng.pek2.redhat.com
+  desiredState:
     interfaces:
     - name: bond10
       description: Bonding eth2 and eth3 for Linux bridge
       type: bond
       state: up
       link-aggregation:
+        mode: balance-rr
+         options:
+          miimon: '140'
         port:
         - eth2
         - eth3
-    - name: br1
-      description: Linux bridge on bond
-      type: linux-bridge
+    - name: bond10.103
+      description: vlan using bond10
+      type: vlan
       state: up
-      bridge:
-        port:
-        - name: bond10
-...
+      vlan:
+         base-iface: bond10
+         id: 103
+      ipv4:
+        address:
+        - ip: 33.103.3.10
```

Comment:
Maybe just use DHCP so we don't have to use specific IPs. In that case, this section would look like:
```
ipv4:
  dhcp: true
  enabled: true
```